libbpf-rs/src/util.rs

```diff
@@ -28,6 +28,7 @@ pub fn c_ptr_to_string(p: *const c_char) -> Result<String> {
         .to_owned())
 }
 
+#[allow(dead_code)] // This might be useful in the future
```

Comment:
Let's just delete this. The logic is not complicated and can easily be added later.